updater/defaults/defaults.go

```diff
@@ -56,9 +56,13 @@ func inner(ctx context.Context) error {
 		return err
 	}
 	updater.Register("ubuntu", uf)
+	df, err := debian.NewFactory(ctx)
+	if err != nil {
+		return err
+	}
+	updater.Register("debian", df)
```

Comment:
considering the constructor is doing:
```go
func NewFactory(ctx context.Context) (*Factory, error) {
	f := &Factory{}
	return f, nil
}
```
do we need to return the error (and subsequently instantiate it in this different way?

Reply:
I'm just making it look like the new, future proofed constructor functions.